wallet-connect/package.json

```diff
@@ -0,0 +1,56 @@
+{
+	"name": "sigmadao-ui",
+	"description": "Sigma DAO UI",
+	"homepage": "https://algobuilder.dev",
+	"license": "mpl-2.0",
+	"engines": {
+		"node": ">=14.11.0",
+		"yarn": ">=3.1.0"
+	},
+	"scripts": {
+		"dev": "vue-cli-service serve",
+		"build": "vue-cli-service build",
+		"lint": "yarn lint:js && yarn lint:prettier",
+		"start": "node .output/server/index.mjs",
+		"lint:js": "eslint --ext \".js,.ts,.vue\" --ignore-path .gitignore .",
+		"lint:prettier": "prettier --check .",
+		"lint:fix": "prettier --write --list-different . && yarn lint:js --fix",
+		"upgrade:dependency": "npx yarn-upgrade-all",
+		"test": "echo 'no test'"
+	},
+	"dependencies": {
+		"@algo-builder/algob": "^3.2.0",
+		"@algo-builder/web": "^3.2.0",
+		"algosdk": "^1.14.0",
+		"core-js": "^3.21.1",
+		"crypto-browserify": "^3.12.0",
+		"pinia": "^2.0.12",
+		"stream-browserify": "^3.0.0",
+		"vue": "^3.2.31",
+		"vue-router": "^4.0.14"
```

Comment:
is vue-router needed?

Reply:
Removed the dependency on this from the template.